ipmof/interpenetration.py

```diff
@@ -325,3 +326,88 @@ def save_extension(sim_par, base_mof, mobile_mof, emap, emap_atom_list, new_stru
     extended_structure['packing_factor'] = packing_factor
 
     return extended_structure
+
+
+def run_interpenetration(base_mof, mobile_mof, emap, atom_list, sim_par, sim_dir):
+    """
+    Interpenetration algorithm for job server.
+    1) Checks Interpenetration
+    2) Gets minimum energy structures
+        - Performs collision check by extending interpenetrating structure
+        - Saves requested structure files
+    """
+    summary, new_structures = run_interpenetration(sim_par, base_mof, mobile_mof, emap, atom_list)
+
```

Comment:
I really, really, really, doubt that this works. And if it does, it shouldn't. ;)

def run_interpenetration...
summary, new_structures = run_interpenetration(sim_par...